protection-art-collection.md

```diff
@@ -0,0 +1,12 @@
+| Title               | Protection Art Collection |
+| Author              | OfficialCryptos |
+| Featured text       |Decred: Building Trust through Bulletproof Blockchain Security|
+
+Thesis statement for art collection - What the article is about, and what is it aiming to prove
+
+# Article Title
+
+Art Piece 1 + description
```

Comment:
Decred's hybrid system (pow/pos) + its treasury, makes it nearly indestructible, like a tardigrade:
'The most 'indestructible' animals on Earth, tardigrades can survive up to 30 years without food, live in volcanoes, and endure the vacuum of space. Researchers say they could even survive an asteroid impact like the one that led to the extinction of the dinosaurs.'
decred tardigrade
![decred_tardigrade](https://github.com/PhoenixGreen/Cypherpunktimes-articles/assets/158556488/0aa94c44-c720-4043-a60f-b425ff59d496)